data_processors/pipeline/orchestration/tests/test_umccrise_step.py

```diff
@@ -201,7 +209,8 @@ def test_prepare_umccrise_jobs(self):
         """
 
         # don't have a good wgs_tumor_normal run in DEV yet, this is from PROD
-        somatic_workflow_id = "wfr.f472a00d6f45421bb8637e87531e2c66"
+        # FIXME - added in a somatic+germline workflow run from dev
+        somatic_workflow_id = "wfr.5616c72c82e442f78f0f9f0d6441219e"
```

Comment:
Do you happen to run umccrise; based on output of this somatic+germline workflow run output? If so, could you pointer wfr id pls?

Reply:
Doing this now

Reply:
Workflow run id is wfr.f93fcc7d267b43b0a6c5856ccf3e0134